cmd/podman/main.go

```diff
@@ -40,8 +43,32 @@ func main() {
 
 	if filepath.Base(os.Args[0]) == registry.PodmanSh ||
 		(len(os.Args[0]) > 0 && filepath.Base(os.Args[0][1:]) == registry.PodmanSh) {
+		if !rootless.IsRootless() {
+			fmt.Println("podmansh not supported for root user")
```

Comment:
I think we should make this explicit in the man pages.

Reply:
added